playbooks/bootstrap-node.yml

```
@@ -0,0 +1,139 @@
+      ansible.builtin.copy:
+        content: "dsom_root={{ dsom_root | default('/opt/dsom-persistence') }}\n"
+        dest: /etc/um-elastic-soc.conf
+        owner: root
+        group: root
+        mode: '0644'
+      ignore_errors: yes
+
+    - name: "OS Tuning | Apply setup_os Role"
+      ansible.builtin.include_role:
+        name: setup_os
+      ignore_errors: yes
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)